app/models/ticket.rb

```diff
@@ -26,16 +30,40 @@ class Ticket < ApplicationRecord
   validates_with ImageAttachmentsValidator,
                  if: ->(ticket) { ticket.description_attachments.any? }
 
+  validates_with ImageAttachmentsValidator,
+                 if: lambda { |ticket|
+                   ticket.resolution_attachments &&
+                     ticket.resolution_attachments.any?
+                 }
+
   scope :most_recent, -> { order(created_at: :desc) }
 
   def description_attachments
     description.body.attachments
   end
 
+  def resolution_attachments
+    resolution&.body&.attachments
```

Comment:
Чому тут використовується `&`, а в `description_attachments` ні?
```suggestion
    resolution.body.attachments
```